src/client/client4.js

```diff
@@ -2372,6 +2377,13 @@ export default class Client4 {
     // Client Helpers
 
     doFetch = async (url, options) => {
+        if (!this.online) {
```

Comment:
Shouldn't this be in `doFetchWithResponse`?

Reply:
I'd just expect this check to cover all fetch calls made by Client4. Having it not cover a few of them seems weird to me

Reply:
Sounds fair.  I'll test to make sure everything looks good moving to ` doFetchWithResponse`